examples/controls_example/public/react_controls/control_panel.tsx

```diff
@@ -102,6 +124,8 @@ export const ControlPanel = <ApiType extends DefaultControlApi = DefaultControlA
 
   return (
     <EuiFlexItem
+      ref={setNodeRef}
+      style={style}
       grow={grow}
       data-control-id={api?.uuid}
```

Comment:
uuid is now passed in. How about using `uuid` instead of `api?.uuid`? What is `data-control-id` attribute used for?

Reply:
We use `data-control-id` for pretty much all of our control functional tests (via the controls page object) - not sure why we chose to use this attribute instead of just `data-test-subj` but this is legacy so not sure we want to change it. Replaced `api.uuid` with `uuid` in 22de1695a8aeead327fa9e796b71238bfb9e0bfa